Orange/widgets/unsupervised/owkmeans.py

```diff
@@ -315,7 +315,7 @@ def send_data(self, row=None):
         else:
             km = self.km
         if not self.data or not km:
-            self.send("Data", None)
```

Comment:
Avoid errors like that by not [stringly typing](http://c2.com/cgi/wiki?StringlyTyped) and instead using [simple `enum`s](https://github.com/biolab/orange3-text/blob/master/orangecontrib/text/widgets/owlda.py#L14). Python doesn't catch that invalid class member at its compile time, but some advanced IDEs might.